tests/spread/lib/install-slices

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
```

Comment:
why is this script in a `lib` folder and not in a `bin` one?

Reply:
it's to be aligned with the tests structure adopted for Charmcraft (e.g. https://github.com/canonical/charmcraft/tree/a44c03ee7ba42460bdd2f2fe72f3d25bddcbd80e/charmcraft/templates/init-simple/tests/spread/lib).

In the end, this doesn't matter much cause the path is added to `PATH` in spread.yaml.